test/notebooks/test_notebooks.py

```diff
@@ -15,76 +16,140 @@
 from syft import TorchHook
 from syft.workers.websocket_server import WebsocketServerWorker
 
-# buggy notebooks
-exclusion_list_basic = [
+# lets start by finding all notebooks currently available in examples and subfolders
+all_notebooks = [Path(n) for n in glob.glob("examples/tutorials/**/*.ipynb", recursive=True)]
+
+# buggy notebooks with explanation what does not work
+exclusion_list_notebooks = [
+    # Part 10 needs either torch.log2 to be implemented or numpy to be hooked
     "Part 10 - Federated Learning with Secure Aggregation.ipynb",
+    # Part 13b and c need fixing of the tensorflow serving with PySyft
     "Part 13b - Secure Classification with Syft Keras and TFE - Secure Model Serving.ipynb",
     "Part 13c - Secure Classification with Syft Keras and TFE - Private Prediction Client.ipynb",
-]
-
-exclusion_list_advanced = [
+    # This notebook is excluded as it needs library code modification which I might add later on
     "Build your own tensor type (advanced).ipynb",
     "Federated Recurrent Neural Network.ipynb",
 ]
 
+exclusion_list_folders = [
+    "examples/tutorials/websocket",
+    "examples/tutorials/advanced/Monitor_Network_Traffic",
+    "examples/tutorials/advanced/websockets-example-MNIST-parallel",
+]
+
+# remove known buggy notebooks and folders that should be excluded
+not_excluded_notebooks = []
+for n in all_notebooks:
+    if n.name in exclusion_list_notebooks:
+        continue
+    elif str(n.parent) in exclusion_list_folders:
+        continue
+    else:
+        not_excluded_notebooks.append(n)
+
 
 def test_notebooks_basic(isolated_filesystem):
     """Test Notebooks in the tutorial root folder."""
     notebooks = glob.glob("*.ipynb")
     for notebook in notebooks:
-        if Path(notebook).name in exclusion_list_basic:
-            continue
-        res = pm.execute_notebook(
-            notebook, "/dev/null", parameters={"epochs": 1, "n_test_batches": 5}, timeout=300
-        )
-        assert isinstance(res, nbformat.notebooknode.NotebookNode)
+        print(notebook)
```

Comment:
extra print here

Reply:
Maybe you can to keep the print statement but only is the tutorials is not excluded